usr/local/sbin/entrypoint.d/05-deskpro-cloud.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+#######################################################################
+# This source handles setup steps for Deskpro Cloud.
+#######################################################################
+
+function deskpro_cloud_main() {
+  if [ -z "$DESKPRO_CLOUD_MODE" ]; then
+    boot_log_message TRACE "[cloud] Not running in Cloud mode, skipping cloud setup."
+    return
+  fi
+  echo "set nocompatible" >> /etc/vim/vimrc.local
```

Comment:
Can we just add this file all the time? i.e. it doesn't seem like it would be cloud specific, so we can get rid of this cloud .sh file entirely